packages/app/src/cli/services/info.ts

```diff
@@ -196,13 +194,15 @@ class AppInfo {
     return `\n${string.linesToColumns(details)}`
   }
 
-  invalidExtensionSubSection(extension: UIExtension | FunctionExtension | ThemeExtension) {
+  invalidExtensionSubSection(extension: UIExtension | FunctionExtension | ThemeExtension): string {
+    const error = this.app.errors?.getError(extension.configurationPath)
+    if (!error) return ''
     const details = [
-      [`📂 ${UNKNOWN_TEXT}`, path.relative(this.app.directory, extension.directory)],
+      [`📂 ${extension.configuration?.type}`, path.relative(this.app.directory, extension.directory)],
       ['     config file', path.relative(extension.directory, extension.configurationPath)],
     ]
-    const error = this.formattedError(this.app.errors!.getError(extension.configurationPath)!)
-    return `\n${string.linesToColumns(details)}\n${error}`
+    const formattedError = this.formattedError(error)
+    return `\n${string.linesToColumns(details)}\n${formattedError}`
```

Comment:
Follow on:

Since this was to fix an existing bug, we should probably have a unit test to cover future regressions

Reply:
https://github.com/orgs/Shopify/projects/5369/views/2